util/interface.js

```diff
@@ -81,6 +81,7 @@ const resolveInterfaceName = type => {
 
     case 'Float':
     case 'Integer':
+	case 'Int':
```

Comment:
additionally, can probably remove the `Integer` Type, I think this was just a typo 😢